test/e2e_node/podresources_test.go

```diff
@@ -568,6 +571,48 @@ func podresourcesGetAllocatableResourcesTests(ctx context.Context, cli kubeletpo
 	}
 }
 
+func podresourcesGetTests(ctx context.Context, f *framework.Framework, cli kubeletpodresourcesv1.PodResourcesListerClient) {
+	var err error
+	ginkgo.By("checking the output when no pods are present")
+	_, err = cli.Get(ctx, &kubeletpodresourcesv1.GetPodResourcesRequest{PodName: "test", PodNamespace: f.Namespace.Name})
+	framework.ExpectError(err, "pod not found")
```

Comment:
shouldn't we check we get an empty response?

Reply:
I will add